src/vs/workbench/services/host/electron-sandbox/nativeHostService.ts

```diff
@@ -193,8 +195,21 @@ class WorkbenchHostService extends Disposable implements IHostService {
 
 	//#region Screenshots
 
-	getScreenshot(): Promise<VSBuffer | undefined> {
-		return this.nativeHostService.getScreenshot();
+	getScreenshot(rect?: IRectangle): Promise<VSBuffer | undefined> {
+		return this.nativeHostService.getScreenshot(rect);
+	}
+
+	async getElementData(offsetX: number, offsetY: number, token?: CancellationToken): Promise<IElementData | undefined> {
+		if (token) {
```

Comment:
`if(token)` is questionable? Just have it default to `CancellationToken#None` instead